plugins/modules/lvol.py

```diff
@@ -117,24 +117,24 @@
   ibm.power_aix.lvol:
     vg: test1vg
     lv: test1lv
-    strip_size: 64M
-- name: Create a logical volume of 32M with disks testdisk1 and testdisk2
+    num_of_logical_parititions: 64M
```

Comment:
Why do the examples using strip_size were modified with examples of the new num_of_logical_partitions attribute?

Reply:
the original documentation was misleading.
strip_size parameter is used if you want to create a `striped` LV
but not all LV are `striped` LV, hence strip_size is not a required parameter
however in the examples `Create a logical volume of 32M with disks testdisk1 and testdisk2`
the size `32M` refers to size of the LV and not strip size.
hence, if the user wants to specify the size of the LV they should really use the
`num_of_logical_partitions` parameter instead of `strip_size`